modules/ROOT/pages/sap/sap-connector-studio.adoc

```diff
@@ -0,0 +1,83 @@
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. In Exchange, select *All assets* and search for "SAP".
+. Select the Mule 4 SAP Connector and click *Add to project*.
+. Follow the prompts to install the connector.
+
+== Configure the Connector
+
+If needed, you can create a global SAP element if you need more than one SAP connector instance in your Mule project.
+
+The SAP connector object holds the configuration properties that allow you to connect to the SAP server.
+
+To create a configuration for an SAP connector, complete the following steps:
+
+. Go to Global Elements section and click *Create*.
+. Select either *SAP Inbound* or *SAP Outbound* in the *Connector Configuration* section.
+. In the *General* tab pane, enter the required parameters for defining an SAP connection, which your SAP system administrator can supply.
+
+The SAP *Global Element Properties* allows you to define connection properties as well as to add the required SAP dependencies to your project.
+
+For ease of use, the SAP connector only shows the most common properties as connector parameters. To configure a property that is not listed in the *Properties* panel, see the xref:sap/sap-connector-config-topics.adoc[Define Extended Properties] topic.
+
+The minimum required attributes to define are:
+
+[[gepparms]]
+[%header,cols="30s,70a"]
+|===
+|Field |Description
+|Application Server Host | SAP endpoint
+|Username | Username of an authorized SAP user.
+|Password | Password credential of an authorized SAP user.
+|System Number | System number used to connect to the SAP system.
+|Client| The SAP client ID (usually a number) used to connect to the SAP system.
+|Login Language | The language to use for the SAP connection. For example, EN for English.
+|===
+
+As a best practice, use property placeholder syntax to load the credentials in a more simple and reusable way.
+
+Click *Test Connection* to verify that the connection to the SAP instance is correct. If the credentials are correct you should see the `Test Connection Successful` message.
+
+== SAP Libraries
+
+See xref:sap/sap-connector-config-topics.adoc[Additional Configuration Information] for how to install required SAP libraries, how to define extended properties, and SAP JCO notes.
+
+== Configure an Input Source
+
+Configure an input source for this connector such as a connector operation, an HTTP Listener, or Scheduler.
+
+If you want your connector to start when a document or function changes in your SAP instance, you can use these SAP Connector operations:
+
+* *Document Source*
+* *Function Source*
+
+Both operations have these fields in common: *Configuration* name, *Operation Timeout*, *Gateway Host*, *Gateway Service*, *Program ID*, *Connection Count*, *Targeted Function*, *Transactional Action*, *Primary Node*, *Streaming Strategy*, *Redelivery Policy*, and *Reconnection Strategy*. 
+
+The *Function Source* operation also provides the *Response* payload.
```

Comment:
So...this topic is more of a concept topic about configuring input AND output sources by using the Document Source and the Function Source operations? 
Consider retitling this subtopic for precision. Clarify how this relates to configuration. Provide steps if necessary.